assets/src/containers/ResourcesAccessed.js

```diff
@@ -178,11 +179,14 @@ function ResourcesAccessed (props) {
   useEffect(() => {
     // Fetch data once all the setting data is fetched
     if (dataControllerLoad === 2 && !(resourceTypeFilter.length === 0)) {
-      const dataURL = `/api/v1/courses/${courseId}/resource_access_within_week?week_num_start=${weekRange[0]}&week_num_end=${weekRange[1]}&grade=${resourceGradeFilter}&resource_type=${resourceTypeFilter}`
+      const dataURL = `/api/v1/courses/${courseId}/resource_access_within_week/?week_num_start=${weekRange[0]}&week_num_end=${weekRange[1]}&grade=${resourceGradeFilter}&resource_type=${resourceTypeFilter}`
       const fetchOptions = { method: 'get', ...defaultFetchOptions }
       fetch(dataURL, fetchOptions)
         .then(handleError)
-        .then(res => res.json())
+        .then((response) => {
+          setResourcesLimit(response.headers.get("Resources-Limit"))
+          return response.json()
+        })
```

Comment:
The `resourcesLimit` is not likely to change between sessions (or maybe ever once set), so I'm thinking this value should be added to the MyLA globals context processors, processed in `globals.js`, and passed down the component tree as a `prop`. Then we don't need the extra React state and this header. What do you think?

Reply:
I started with globals, but the backend is limiting this per course and globals is for everything. Some courses may not be limited and others might. If someone was enrolled in multiple courses and we wanted to display the specific limit to the student and it was in globals we'd have to do some calculations in the frontend rather than just passing the data from the backend. 

For instance if we passed the limit of 100, the frontend would have to see if there were 100 results and print a message that it was limited, otherwise it would print the total results. So there'd be some logic there.

There's also a small chance say if someone viewed this page before the reload and then again after that there would be more there that wouldn't be updated in the globals. Or since this is set in the admin, an admin could change the limit and get this de-synced. I figured passing it from the backend each time eliminates all these edge cases. 

I think we could also use these headers in the future, and was somewhat inspired by the Canvas pagination headers for trying this out.

Reply:
Okay, yeah, I thought of the edge cases you alluded to, but you are adding complexity in other ways. I'm okay with this approach if you are.